lambda/translcore.ml

```diff
@@ -1543,27 +1386,8 @@ and transl_function ~in_new_scope ~scopes e alloc_mode param arg_mode arg_sort r
   in
   let attr = default_function_attribute in
   let loc = of_location ~scopes e.exp_loc in
-<<<<<<< HEAD
   let body = if region then maybe_region_layout return body else body in
   let lam = lfunction ~kind ~params ~return ~body ~attr ~loc ~mode ~region in
-||||||| merged common ancestors
-  let lam = Lfunction{kind; params; return; body; attr; loc} in
-  Translattribute.add_function_attributes lam e.exp_loc e.exp_attributes
-=======
-  let lam = lfunction ~kind ~params ~return ~body ~attr ~loc in
-  let attrs =
```

Comment:
This bit looks important. Is there a reason to drop?

Reply:
It's done at the top of the function internally.

Reply:
(We backported this and changed it slightly to make it work with zero-alloc attributes.)